lib/rails/generators/channel/templates/channel.coffee.erb

```diff
@@ -0,0 +1,3 @@
+App.<%= channel_name %> = App.cable.subscriptions.create "<%= channel_class %>",
```

Comment:
All of Cable's JavaScript is written in CS so it's fine to generate .coffee for now.